### PR TITLE
tooltip to info button

### LINKED ISSUE #232: replace tooltips with info button

### PR DESCRIPTION
**Pull request by martin-lenweiter**

### CODE REVIEW

**Review by Nitwix — CHANGES_REQUESTED** [review]
I'd like to see the duplicate code refactored, but other than that, good job!

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
~~We can nuke that then~~

My bad I was running on main

**Review by bbjubjub2494 — APPROVED** [review]

File: app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningSettingsFragment.kt [hunk 1]
```diff
@@ -84,6 +77,31 @@ class LearningSettingsFragment : Fragment() {
...
+                    .setTitle(getString(R.string.info))
+                    .setMessage(getString(R.string.displayLearningSettingsInfo))
+                    .show()
+                true
+            }
+            else -> {
+                true
+            }
+        }
+    }
```
[the remark this answers is not in the document]
Comment (martin-lenweiter) [reply app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningSettingsFragment.kt]: sure I'll do that.

File: app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/FragmentOptionsUtil.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package com.github.HumanLearning2021.HumanLearningApp.view
+
+import android.app.AlertDialog
+import android.content.Context
+import android.view.MenuItem
+
+object FragmentOptionsUtil {
```
Comment (Nitwix) [inline app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/FragmentOptionsUtil.kt:10]: Ideally it would be good to add kdoc

**Review by Nitwix — APPROVED** [review]
See comment about missing kdoc, but good otherwise.

**Pull request merged**